Description: Do not include grgsm_setup_freedesktop in binary package
 Stop installing the grgsm_setup_freedesktop script in /usr/libexec/.
 Its only purpose is to install and uninstall the desktop file, and
 this do not belong in the installed system.
Author: Petter Reinholdtsen <pere@hungry.com>
Forwarded: no
Reviewed-By: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2017-09-02

--- gr-gsm-0.41.1.orig/apps/apps_data/CMakeLists.txt
+++ gr-gsm-0.41.1/apps/apps_data/CMakeLists.txt
@@ -34,8 +34,4 @@ if(UNIX AND HAVE_XDG_UTILS)
         ${CMAKE_CURRENT_SOURCE_DIR}/grgsm_setup_freedesktop.in
         ${CMAKE_CURRENT_BINARY_DIR}/grgsm_setup_freedesktop
     @ONLY)
-    install(
-        PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/grgsm_setup_freedesktop
-        DESTINATION ${GR_PKG_LIBEXEC_DIR} COMPONENT "gr-gsm"
-    )
 endif(UNIX AND HAVE_XDG_UTILS)
